lib/interface_quda.cpp

```diff
@@ -1770,6 +1772,33 @@ namespace quda {
   }
 }
 
+void distanceReweight(ColorSpinorField &b, QudaInvertParam &param, bool inverse)
```

Comment:
Can this function be moved to `spinor_reweight.cu`?  It's not an interface function, and would be better placed there.

Reply:
As discussed on Slack, this function is an utility function just like `massRescale`, leave it here for now.